installscripts/cookbooks/jenkins/files/default/jazz-installer-vars.json

```diff
@@ -115,10 +120,29 @@
     },
     "APIGEE": {
         "ENABLED": false,
+        "APIGEE_CRED_ID": "{APIGEE_CREDS}",
+        "MAVEN": "apache-maven-3.3.9",
+        "BUILD_VERSION": "1.0",
+        "USE_SECURE": false,
         "API_ENDPOINTS": {
-            "PROD": "{APIGEE_PROD_API}",
-            "STG": "{APIGEE_STG_API}",
-            "DEV": "{APIGEE_DEV_API}"
+          "PROD": {
+            "MGMT_ENV": "prod",
+            "MGMT_ORG": "{ORG_NAME}",
```

Comment:
Will the replacement logic be able to differentiate between this ORG_NAME and the ORG_NAME tokens in the STG and DEV blocks?

Reply:
OK updated